alternatingCase/alternatingCase.js

```diff
@@ -0,0 +1,16 @@
+require("../toUpper/toUpper");
```

Comment:
Change this to a single quote. Look at Airbnb Javascript style guide. Make this correction across your code.